.github/workflows/golang-security-scan.yml

```
@@ -99,21 +104,26 @@ jobs:
           echo "REPORT_NAME=govulncheck-${{ matrix.checkout-ref == '' && '1' || '2' }}" >> $GITHUB_ENV
           echo "REPORT_PATH=govulncheck-report-${{ matrix.checkout-ref == '' && '1' || '2' }}.txt" >> $GITHUB_ENV
 
+      - name: Set up Go for govulncheck
+        uses: actions/setup-go@v5
+        with:
+          go-version: "stable"
+          cache: false
+
+      - name: Install govulncheck
+        run: go install golang.org/x/vuln/cmd/govulncheck@latest
```

I'm puzzled as to why there is a `govulncheck-version` input added above, but on this line, it isn't using that new input. Is this supposed to be something along these lines?
```suggestion
        run: go install golang.org/x/vuln/cmd/govulncheck@{{$govulncheck-version}}
```